lib/node-arguments.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const trim = require('lodash/trim');
+
+// Parse formats:
+// --param=--parameter --abc
+//         |_this____|
+// --param=" --arg1 --arg2 param " --abc
+//         |_this part___________|
+function parseCliParameter(str) {
+	str = str[0].startsWith('"') && str.endsWith('"') ? str.slice(1, str.length - 1) : str;
+	return trim(str)
+		.split(' ');
+}
+
+function normalizeNodeArguments(nodeArguments) {
+	const parsed = Array.isArray(nodeArguments) ? nodeArguments : parseCliParameter(nodeArguments);
```

Comment:
If the value comes from a configuration file, we should enforce that it's an array.

When it comes from the CLI, begrudgingly, I agree we need to parse it. Is there an npm package that can take care of this for use?